### PR TITLE
Hope2 (updated the code associated with the website files, update the readme files for guidance)

### PR DESCRIPTION
**Pull request by AjeshStavir**
Updated the region associated with aws as well as the table name to "shared-list".
Updated quick-shopping-list with Shared List.
Updated email id to hr@stavir.com.
Updated README.me with the configuration made inside AWS (lambda, API Gateway, S3, Cloudfront).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,3 +34,47 @@ Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protrac
...
+
+## Lambda functions.
+
+ Created three lambda functions from the code base namely:-
+ getshoppinglist, updateshoppinglist, CORS_function from the code base provided through github.
+ 
+ Getshoppinglist for generating an id and for calling with that id while updateshoppinglist for update the list with that given id and CORS_function to handle HTTP requests(get,put,options) and to allow CORS( 
+ Cross Origin Resource Sharing). 
+ Gave necessary IAM permissions in the form of role to the lambda functions so as toIntegrate it with the dynamodb table.
+ Testing the lambda function with appropriate payload (payload not for CORS_function)was done successfully.
```
Comment (skStavir) [inline README.md]: Was done successfully does not sound right here. Specify this as an instruction for others to follow

File: README.md [hunk 2]
```diff
@@ -34,3 +34,47 @@ Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protrac
...
+
+## API gateway Integration
+
+ API Gateway enables to connect APIs to various backend services, such as AWS Lambda functions, Amazon EC2 instances, or other HTTP endpoints. For connecting lambda functions to APIs we have created an API 'sharedlist'.
+
+created the required routes for the api according with the lambda functions so here it is GET ( to get an id), GET {id} ( to get the id by calling a particular id), PUT( to update the shopping list associated with a particular id) and OPTIONS ( To describe the communication options for the target resource). 
+
+So Get and GET {id} was integrated with getshoppinglist lambda function and PUT on to updateshoppinglist function while OPTIONS on to CORS_function through
+‘Manage integration’.
+
```
Comment (skStavir) [inline README.md:69]: Add some screenshots here

File: README.md [hunk 3]
```diff
@@ -34,3 +34,47 @@ Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protrac
...
+Access-control-allow-origin is configured with https://stavir.com
+Access-control-allow-headers is configured with content-type.
+Access-control-allow-methods with GET,PUT,OPTIONS.
+
+The api was tested through POSTMAN tool and ensured its successful availability.
+
+## S3 Bucket configuration
+
+After successful build of the application soon after making changes within the code in angular js format, using the command ‘ng build’, the files generated within the ‘dist’ folder has to be uploaded to the ‘stavir’ S3 bucket and made public with the changes.
```
Comment (skStavir) [inline README.md]: Specify the complete  ng build comment for production

File: README.md [hunk 4]
```diff
@@ -34,3 +34,47 @@ Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protrac
...
+
+A file named shoppingitems.json was also uploaded to make the suggestion work whenever a user types the first or two letter(s) of the item.
+
+The path to the index was given as ‘stavir.com/tools/sharedlist’.
```
Comment (skStavir) [inline README.md]: Do not specify was given as. Maybe you can specify as an example

**Review by skStavir — COMMENTED** [review]
Please see the individual comments on read me and fix it

File: README.md (hunk 2 above)
Comment (AjeshStavir) [reply README.md:69]: Unable to pin screenshot inside the file.

**Review by skStavir — DISMISSED** [review]

**Review by skStavir — APPROVED** [review]

**Pull request merged**